lms/djangoapps/instructor/access.py

```diff
@@ -44,6 +44,22 @@ def list_with_level(course, level):
     return ROLES[level](course.id).users_with_role()
 
 
+def list_with_level_from_course_key(course_key, level):
```

Comment:
The second of two slightly duplicated functions introduced in this PR. The original function passed in a course but only _really_ needs a course_key/course_id. However, there are a quite a few places this function is being used and I didn't want to refactor all of this in one go.

We can either finish the refactor of this method later, or assign the subtask I created to the team who owns the instructor django app to finish themselves (if desired). I need this function to remove the modulestore/course dependency from the generation_handler file though.